bats/tests/pyrsia_cli.bats

```diff
@@ -79,6 +79,44 @@ setup() {
   assert_output --partial '7888'
 }
 
+@test "Testing 'pyrsia config edit' CLI, check if the config can be changed." {
```

Comment:
@ihcomega56 There are two features we should implement as a part of this test:
1) If this change impacts the local development environment then we should reset the Pyrsia configuration to the default values when testing is over. The "reversal" cannot be part of the test since it's not guaranteed to be executed but rather should be part of the method which is executed when all tests in the file are completed.
2) Please correct me if I'm wrong but I don't see any code which would check if the config contains the edited values. For example, if you edit a port to 9999 then check if the node config contains "9999". As far as I understand your code only checks if there are errors when you update the config, but no checks if the config content is correct.